### PR TITLE
A few small changes to add --region and --profile

### PR DESCRIPTION
**Pull request by robert-john-small**
A few small changes to add --region, --profile and fix a bug where AWS reports -1 as the protocol.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Useful to keep track of the firewall changes in git.
 Can also be used as a backup in case you lose some rules on EC2.
 
 Usage:
-    ec2-security-groups-dumper --json
-    ec2-security-groups-dumper --csv
+    ec2-security-groups-dumper --json [--region=<region>] [--profile=<profile>]
```
Comment (lra) [inline README.md]: Can you instead add those as options and add an `[options]` to each usage, except `--help`?
See `[options]` on http://docopt.org/

File: ec2_security_groups_dumper/main.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
 Can also be used as a backup in case you lose some rules on EC2.
 
 Usage:
-    ec2-security-groups-dumper --json
-    ec2-security-groups-dumper --csv
+    ec2-security-groups-dumper --json [--region=<region>] [--profile=<profile>]
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: same as above

File: ec2_security_groups_dumper/main.py [hunk 3]
```diff
@@ -93,7 +93,9 @@ def as_dict(self):
 
 class Firewall(object):
 
-    def __init__(self):
+    def __init__(self, region=None, profile=None):
```
Comment (lra) [inline ec2_security_groups_dumper/main.py]: Can you add a docstring and describe what the region and profile args are for? Also assert that they are of the correct type if not None.

File: ec2_security_groups_dumper/main.py (hunk 2 above)
Comment (robert-john-small) [inline ec2_security_groups_dumper/main.py]: Fixed, see above.

File: ec2_security_groups_dumper/main.py (hunk 3 above)
Comment (robert-john-small) [inline ec2_security_groups_dumper/main.py]: Actually, needs asserting still.

Comment (robert-john-small) [inline ec2_security_groups_dumper/main.py]: Fixed.

**Comment by lra** [comment]
Thanks

File: README.md (hunk 1 above)
Comment (selvakann) [inline README.md]: how get outbound rules in ec2-security-groups-dumper.

Comment (selvakann) [inline README.md]: i am new to python boto

**Pull request merged**